src/main/java/org/shipkit/auto/version/NextVersionPicker.java

```diff
@@ -33,8 +33,20 @@ String pickNextVersion(Optional<Version> previousVersion, VersionConfig config,
         }
 
         if (!config.getRequestedVersion().isPresent()) {
-            String tag = runner.run("git", "describe", "--tags").trim();
+            String tag;
             String result;
+
+            try {
+                tag = runner.run("git", "describe", "--tags").trim();
+            } catch (Exception e) {
+                result = "0.0.1-SNAPSHOT";
+                log.info("Process 'git describe --tags' exited with non-zero exit value. Assuming there are no tags. " +
+                        "Run with --debug for more.");
+                log.debug("Ignored exception from 'git describe --tags'. Assuming there are no tags.", e);
+                explainVersion(log, result, "couldn't run 'git describe --tags'");
```

Comment:
"couldn't run 'git describe --tags'" -> "couldn't run 'git describe --tags' (assuming there are no tags)"

Reply:
@mockitoguy thanks, it's done